assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/utilities.js

```diff
@@ -0,0 +1,21 @@
+import path from 'ramda/src/path';
+import { amountsMatch } from '../../../../../application/utilities/money';
+export const isEqual = (value, prev) => prev === null || prev === undefined || value === prev;
```

Comment:
Maybe https://ramdajs.com/docs/#equals ?

Reply:
in my current PR i have converted this to TS and added the following jsdoc to describe what the function does:

```
returns true if the two supplied values are equal after parsing as floats (or previous value is null or undefined)
```

so `R.equals` won't add anything more here, because I would still have to check if `prev` was `null` or `undefined`

Reply:
Thanks, it makes sense.